setup.py

```diff
@@ -3,127 +3,21 @@
-            ),
-            shell=True,
-        )
-    sys.path.insert(0, local_path)
-
-import openapiart
-
-
 # read long description from readme.md
 base_dir = os.path.abspath(os.path.dirname(__file__))
 with open(os.path.join(base_dir, "readme.md")) as fd:
     long_description = fd.read()
 
-if USE_MODELS_DIR is not None:
-    API_FILES = [
-        os.path.join(USE_MODELS_DIR, "api", "info.yaml"),
-        os.path.join(USE_MODELS_DIR, "api", "api.yaml"),
-    ]
-elif USE_MODELS_BRANCH is not None:
-    local_path = "models"
-    if not os.path.exists(local_path):
-        subprocess.check_call(
-            "git clone {} && cd {} && git checkout {} && cd ..".format(
-                MODELS_REPO, local_path, USE_MODELS_BRANCH
-            ),
-            shell=True,
-        )
-    API_FILES = [
-        os.path.join(local_path, "api", "info.yaml"),
-        os.path.join(local_path, "api", "api.yaml"),
-    ]
-else:
-    # download openapi.yaml
-    response = requests.request("GET", OPENAPI_YAML_URL, allow_redirects=True)
-    assert response.status_code == 200
-    with open(os.path.join("openapi.yaml"), "wb") as fp:
-        fp.write(response.content)
-    API_FILES = ["openapi.yaml"]
-
-openapiart.OpenApiArt(
-    api_files=API_FILES,
-    protobuf_name=model_protobuf_name,
-    artifact_dir="artifacts",
-    extension_prefix=pkg_name,
-    generate_version_api=True,
-).GeneratePythonSdk(package_name=pkg_name, sdk_version=version).GenerateGoSdk(
-    package_dir="github.com/open-traffic-generator/snappi/%s" % go_pkg_name,
-    package_name=go_pkg_name,
-    sdk_version=version,
-).GenerateGoServer(
-    module_path="github.com/open-traffic-generator/snappi/%s" % go_pkg_name,
-    models_prefix=go_pkg_name,
-    models_path="github.com/open-traffic-generator/snappi/%s" % go_pkg_name,
-).GoTidy(
-    relative_package_dir=go_pkg_name
-)
-
-if os.path.exists(pkg_name):
-    shutil.rmtree(pkg_name, ignore_errors=True)
-
-# remove unwanted files
-shutil.copytree(os.path.join("artifacts", pkg_name), pkg_name)
-shutil.copyfile(
-    os.path.join("artifacts", "requirements.txt"),
-    os.path.join(base_dir, "pkg_requires.txt"),
-)
-shutil.copyfile(
-    os.path.join(base_dir, "artifacts", model_protobuf_name + ".proto"),
-    os.path.join(base_dir, model_protobuf_name + ".proto"),
-)
-shutil.copyfile(
-    os.path.join(base_dir, "artifacts", model_protobuf_name + ".proto"),
-    os.path.join(
-        base_dir, go_pkg_name, model_protobuf_name, model_protobuf_name + ".proto"
-    ),
-)
-
-for name in os.listdir(pkg_name):
-    if name != "artifacts":
-        path = os.path.join(pkg_name, name)
-        print(path + " will be published")
-
-doc_dir = os.path.join(pkg_name, "docs")
-os.mkdir(doc_dir)
-shutil.move(os.path.join("artifacts", "openapi.yaml"), doc_dir)
-
 with open("models-release", "w") as out:
     out.write("v" + models_version)
+
 install_requires = []
 with open(os.path.join(base_dir, "pkg_requires.txt"), "r+") as fd:
```

Comment:
we should be reading from `snappi/requirements.txt` instead of `pkg_requires.txt` since former will always be present in repo.

Reply:
then we have to move the requirements.txt in snappi folder before dist which in turn will package the snappi code with requirements.txt which is not right I guess